pkg/config/config.go

```diff
@@ -418,6 +444,98 @@ func NewConfig(userConfigPath string) (*Config, error) {
 	return config, nil
 }
 
+func hasLibpodConf() bool {
```

Comment:
I don't believe we need this function, we should just call into the code to get libpod.conf and if it does not exist then just continue on. No reason to check twice